# Ref:
# https://answers.ros.org/question/326628/how-to-control-velocity-of-joints-in-gazebo-using-ros_controllers/

vss_robot_left_controller:
  type: velocity_controllers/JointVelocityController
  joint: roda_esq_joint
  pid:
    p: 100.0
    i: 0.01
    d: 10.0

vss_robot_right_controller:
  type: velocity_controllers/JointVelocityController
  joint: roda_dir_joint
  pid:
    p: 100.0
    i: 0.01
    d: 10.0
